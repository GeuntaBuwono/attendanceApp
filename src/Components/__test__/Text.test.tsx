/**
 * @format
 */

import 'react-native'

import { render } from '@testing-library/react-native'
import Label from 'Components/Text/Text'
import React from 'react'
// Note: test renderer must be required after react-native.

describe('Label', () => {
  it('Render Correctly', async () => {
    const tree = render(<Label>Label Test</Label>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
