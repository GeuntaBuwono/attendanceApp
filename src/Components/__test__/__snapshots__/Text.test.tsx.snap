// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label Render Correctly 1`] = `
<Text
  className="sc-bczRLJ bLKfFM"
  color="#2C2C2B"
>
  Label Test
</Text>
`;
